docs/deep-dives/github.md

```diff
@@ -43,18 +43,12 @@ steps:
 ```
 
 With Docker Compose,
-declare the environment variables and volume mounts in your [Compose file]:
+declare volume mounts in your [Compose file].
+You won't need to declare the environment variables in the compose file anymore since `propagate-environment: true` will ensure the variables defined are propagated to the docker environment.
```

Comment:
Could we include a similar buildkite pipeline example in the same vein as the docker plugin section above?